{{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/content.py

```diff
@@ -6,4 +6,4 @@
 
 @implementer(IHomepage)
 class Homepage(Container):
-  """"""
+  """Homepage content type"""
```

Comment:
There are a hidden exception about no docstring in the content type class that makes default view don't work.